### DISCUSSION TITLE
Support for large file uploads

### DISCUSSION THREAD

**Discussion by blairliikala** [ExpressionEngine/ExpressionEngine#2874] [Feature Requests]
In the new File manager, allow for file uploads that are larger than the post_max_size limit.

Ideally the user can upload a file of any size regardless of server configurations since that is their expectation from using most apps on the Internet.  Chunked uploading is usually the technical solution for this.  Additionally this should help with resuming from network disruptions.  POST should be used as a fallback or files under 200MB.

Currently the options are to use another method such as FTP to get the file to the server, then resync the directory or use Assets.  Both of uses are fairly poor experiences for the user.  The user should just be able to upload files no matter the size s  Alternatively, the File Manager should check the size and reject with a more clear error message that is less technical, such as "the file size is larger than what the server is allowed to accept."

**Comment by netz0** [comment]
Letting your browser overwrite a server limit is not a great idea. There is a reason why limits like post_max_size exist, to avoid exhausting server resources in case of bugs or just plain abuse.
